src/main/java/jackyy/dimensionaledibles/block/BlockCustomCake.java

```diff
@@ -62,11 +62,9 @@ public boolean onBlockActivated(World world,
                                     float hitY,
                                     float hitZ) {
 
-
-
         int dim = getDimension(world, pos);
-        if(!cache.containsKey(dim)) {
-            logger.error("No such dimension: \"{}\"", dim);
+        if(!isDimensionRegistered(dim)) {
+            logger.error("Requested dimension: \"{}\" does not exist. Please verify your configs.", dim);
```

Comment:
Instead of just logging, a chat message should be displayed as well, because I imagine that most players will not have their log open while playing the game, and so will miss the logger warning, leaving them wondering why the cake is not working.

Reply:
Yeah, that's a fair point. I've reworked this such that the server thread will log an error and the client thread will print a message to the user in-game.